txpipe/twopoint.py

```diff
@@ -250,12 +253,14 @@ def add_data_points(self, S, results):
         GAMMAT = sacc.standard_types.galaxy_shearDensity_xi_t
         GAMMAX = sacc.standard_types.galaxy_shearDensity_xi_x
         WTHETA = sacc.standard_types.galaxy_density_xi
+        POS_EXT = "pos_ext"
```

Comment:
Why not buiild a sacc data type for it? you can do something like `GAMMATS = sacc.build_data_type_name(
            "galaxy", ["shear", "Density"], "xi", subtype="ts"
        )` (Example pulled from the self-calibration extension

Reply:
I didn't know you could do this! updated it